src/Forms/DatetimeField.php

```diff
@@ -177,9 +357,15 @@ public function setValue($value, $data = null)
         // If invalid, assign for later validation failure
         $isoFormatter = $this->getISO8601Formatter();
         $timestamp = $isoFormatter->parse($value);
+
+        // Retry without "T" separator
+        if (!$timestamp) {
```

Comment:
See above, getISO8601Formatter() should probably be fixed to ISO_DATETIME, then we won't need a fallback.